src/System.Text.Json/tests/JsonReaderTests.cs

```diff
@@ -293,15 +289,15 @@ private static void ReadJsonHelper(JsonReader jsonReader, ICollection<Members> j
 
         private static Value GetValue(ref JsonReader jsonReader)
         {
-            var value = new Value {Type = (Value.ValueType) jsonReader.GetValueType()};
-            var obj = jsonReader.ReadValue();
+            var value = new Value {Type = (Value.ValueType) jsonReader.GetJsonValueType()};
+            var obj = jsonReader.GetValue();
```

Comment:
Ah, ok. I would just feel much better if you removed this method, but it's ok to do it in a separate commit.